dolphinscheduler-common/src/main/resources/common.properties

```diff
@@ -91,3 +91,5 @@ development.state=false
 # rpc port
 alert.rpc.port=50052
 
+# Url endpoint for zeppelin RESTful API
+zeppelin.rest.url="http://localhost:8080"
```

Comment:
zeppelin is a task plugin for dolphin, I recommend this parameter as a parameter of the zeppelin plugin instead of a public parameter.

Reply:
May I ask where can I add parameter for zeppelin plugin instead of public parameter? Is there anything like task-plugin level `properties` file?

Reply:
> @zhuangchong May I ask where can I add parameter for zeppelin plugin instead of public parameter? Is there anything like task-plugin level properties file?

Why we should have separate configurations for a single task? I think we should get all config in one single config file. It is easy to maintain and also easy for uses. For ex, when we change our task structure(like add provider) we have to move our code from one side to another, if config file alone with task code, maybe they should change the position too. And this will make user confuse for the position change.

Reply:
Maybe we should add a new component named `connection` for the dynamic config task plugin. And in this way, we could not only have task config but also can change config without restart our server

Reply:
> Maybe we should add a new component named `connection`, like Airflow, for the dynamic config task plugin. And in this way, we could not only have task config but also can change config without restart our server

The `connection` idea sounds cool. It will be more convenient for users to change their credentials.

Reply:
Yeah, but we need more discussion about it on our mailing list or other places. And config like your patch it is good for this PR for now